packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1045,6 +1045,12 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
   bool _drawerOpened = false;
   bool _endDrawerOpened = false;
 
+  /// Whether the [Scaffold.drawer] is opened.
+  bool get drawerOpened => _drawerOpened;
+
+  /// Whether the [Scaffold.endDrawer] is opened.
+  bool get endDrawerOpened => _endDrawerOpened;
```

Comment:
A "see also" section that pointed at https://docs.flutter.io/flutter/material/ScaffoldState/openEndDrawer.html would be helpful.

The openEndDrawer() doc should similarly "see also" isDrawerOpen.